### PR TITLE
Upgrade to Scala 2.12

### PR DESCRIPTION
**Pull request by andrusha**
Another shot at Scala version upgrade, just as in #33 and #34. It has to be done eventually.

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 Please visit https://github.com/snowplow/snowplow/wiki/CLA to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: build.sbt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Dependencies._
```
Comment (BenFradet) [inline build.sbt]: can we add the apache 2 header?

**Comment by andrusha** [comment]
@snowplowcla signed

**Comment by andrusha** [comment]
@BenFradet added apache header

**Comment by BenFradet** [comment]
@andrusha sorry, there is something I missed during the first review: could you modify [.travis.yml](https://github.com/snowplow/scala-maxmind-iplookups/blob/develop/.travis.yml#L3-L5) so that the scala versions are the ones you moved to (2.11.12 and 2.12.4)?

**Comment by andrusha** [comment]
@BenFradet my bad, fixed!

**Comment by BenFradet** [comment]
Cool, merging, thanks a lot :+1:

**Pull request merged**